emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java

```diff
@@ -397,9 +401,11 @@ private Builder(final View rootView) {
      * If height is not 0 then this value will be used later on. If it is 0 then the keyboard height will
```

Comment:
These docs are not correct anymore,should be something like "if height is greater than 0".

Reply:
Fixed.